pipeline/ephys.py

```diff
@@ -629,6 +629,21 @@ class WaveformMetric(dj.Part):
         velocity_below=null: float   
         """
 
+
+@schema
+class UnitPassingCriteria(dj.Computed):
+    """ This table computes whether a unit_key passes a selection criteria """
+    definition = """
+    -> Unit
+    ---
+    criteria_passed : bool      #boolean based on criteria
```

Comment:
the comment `boolean based on criteria` is not really helpful there (doesn't really say anything more than what can be inferred from the attribute name), can you improve this a bit to let the users know what this field is about

Reply:
Will do